src/Routing/ContentfulSlugMatcher.php

```diff
@@ -218,13 +218,16 @@ private function getMatchingEntry(string $requestUri)
             $cacheHit->set($entry);
 
             if ($cacheTTL = $this->getCacheTTL()) {
-
                 $cacheHit->expiresAfter($cacheTTL);
             }
 
             if (!$this->isCacheIgnored) {
-                if (method_exists($cacheHit, 'tag')) {
-                    $cacheHit->tag($this->getCacheTags($entry));
```

Comment:
Because of the parsing inside of the client decorator $entry will always be an array. getCacheTags only returns Tags for DynamicEntries.

Reply:
After this removal, you can remove the cache tags trait alltogether.